firebase.json

```diff
@@ -77,6 +77,62 @@
         "source": "/",
         "destination": "/docs/",
         "type": "302"
+      },
```

Comment:
Change them all to 302, just so we make sure it is working.. then we change to 301